### PR TITLE
Updated target table browser

### PR DESCRIPTION
**Pull request by AJK-dev**
* Updated table target browser (layout + new columns + link-outs)
* Replaced hardcoded alerts with soft alerts for interface browser
* Bit of code cleaning (more to be expected)

### CODE REVIEW

File: common/views.py [hunk 1]
```diff
@@ -127,14 +138,28 @@ def getTargetTable():
             t['uniprot'] = p.entry_short()
             t['iuphar'] = p.family.name.replace("receptor", '').strip()
 
+            # Web resource links
+            link_setup = "</span><a target=\"_blank\" href=\"{}\"><span class=\"glyphicon glyphicon-new-window btn-xs\"></span></a>";
+            t['uniprot_link'] = ""
+            t['gtp_link'] = ""
+            uniprot_links = p.web_links.filter(web_resource__slug='uniprot')
```
Comment (esguerra) [inline common/views.py:145]: Ahh! Nice trick to learn.

File: static/home/js/matrix_utilities.js [hunk 2]
```diff
@@ -1,3 +1,5 @@
+/*global Papa,d3,yadcf,showPDBtable,showAlert, _, signprotmat, con_seq, non_interactions,interactions_metadata,interactions,gprot,receptor*/
```
Comment (esguerra) [inline static/home/js/matrix_utilities.js:1]: what is this for?
helps codacy somehow?

**Review by esguerra — APPROVED** [review]
Looks good. 
Nice to learn how to use the slug for weblinks.

File: static/home/js/matrix_utilities.js (hunk 2 above)
Comment (AJK-dev) [reply static/home/js/matrix_utilities.js:1]: Jeps - we might want to exclude this, but it's good way to find errors in calls/variables/functions

**Pull request merged**